docker-compose.yml

```diff
@@ -27,10 +27,10 @@ services:
       && python manage.py loaddata user.yaml
       && python manage.py loaddata support.yaml
       && python manage.py loaddata support_layerfilter_fixtures.yaml
+      && python manage.py loaddata urgent_alerts_fixtures.yaml
```

Comment:
File renamed, but in docker-compose is not renamed.

Reply:
Modified.